### ISSUE TITLE
just an idea

### ISSUE THREAD

**Issue by cicina** [sophron/wifiphisher#401]
Hello, just wondering.. 

it might be useful if you would add an option to copy access point with same channel but different name.
example:
There is wifi named TodNet
I would like to attack their users and create fake AP as usual but I would named it TodNet Upgrade
or something similar, to get attention. Name of the fake AP could be free editable by attacker.

Im trying to do this by using aireplay-ng for deauth and fake AP using an --nJ option in wifiphisher with --essid option. 

Well, not much success so far.. 
Q: is there any strategy to make victims notice bloody fake AP? they got no internet connection for two days but they just won't connect to my AP. 
Maybe deauth not working but I dont think so since I use aireplay-ng for deauth, which shows sent packets.. I dont know..

**Comment by laozi999** [comment]
Be careful with long during deauthentication, especially if you're doing it without permission, it's illegal.

I don't see why you would want a different name to 'attract attention', you are better off using the same name to trick them into connecting, or use 'TodNet ' (with a space) to bypass some network warnings.

You can change the channel of your AP in `/wifiphisher/wifiphisher/constants.py`

**Comment by cicina** [comment]
Well, understand, but still, it would be useful to be able to modify essid. If I do, it will skip jamming. 
I dont know how would I use this yet, but I am sure it could be used somehow.

**Comment by sophron** [comment]
Hi @cicina,

I can't imagine many use cases for this (except maybe from WPS PBC Technique #264). We can probably implement this at some point in the future if more people are interested in this.

**Comment by laozi999** [comment]
You are still able to do this, you will have to manually deauth however.

I like to use mdk3.

This sends deauth packets **AND** disassociation packets to all access points, except for the BSSID(s) listed in a whitelist file:

`mdk3 wlan1 d -w "/path/to/whitelist.txt"`

This sends deauth packets **AND** disassociation packets only to the BSSID(s) listed in a blacklist file:

`mdk3 wlan1 d -b "/path/to/blacklist.txt"`

(you can issue `mdk3 --fullhelp` for more options)

Use the `-nJ` (--nojamming) option in wifiphisher and you can basically appear as any ESSID while deauthing whatever AP you like.

**Comment by laozi999** [comment]
This method could be easily implemented into wifiphisher, and allow users to jam more than just 1 AP.

**Comment by sophron** [comment]
@laozi999 There are many improvements that I have in mind regarding jamming, including some novel techniques.

However, this issue is about spawning an AP and deauthing another one (with different ESSID). A user could possibly use the `--essid` option and run `mdk3`, as you said, but it may be useful to run it from the tool itself.

**Comment by cicina** [comment]
Indeed, guys, we can run it separately. Thanks for quick response. 

@laozi999 
What is the difference if you send deauth AND disassociation packets? 
Am I correct, aireplay-ng sends only deauth packets? 
@sophron indeed, it may be usefol to run it from tool itself however I know there is more important things to implement. 
Thanks

**Comment by laozi999** [comment]
@cicina 
Basically, mdk3 kicks everyone off the AP, while with aireplay-ng you can specify a single client.

**Comment by cicina** [comment]
Thanks I have been playing around with mdk3 its rly nice. Haven't heard of it before.

**Comment by laozi999** [comment]
Also a very cool feature about mdk3 is that it will stream deauthentication and disassociation packets to the target on all channels, whereas with aireplay-ng, you have to set your interface to the right channel before you can successfully deauth the target.

**Comment by cicina** [comment]
Noticed that! Its really annoying if you dont want to focus on one specific ap and want to attack several of them and try your luck.
Wifiphisher handle this brilliantly. However i would preffer to see progress or just sent packets to see if deauth running successfully or not. This thing is missing as I was attacking my router just for testing purposes and I found out that not always deauth works. It would be good to see real time how it is doing. :)

**Comment by laozi999** [comment]
If you run mdk3 on a target, and you are in reasonable range or have a strong enough antenna, your deauth packets should be effective and continuous for the duration that you are running it

**Comment by mcocigra** [comment]
From my experience people don't trust things they don't know. I haven't used wifiphisher for a few months and the new enhancements are good bit there are also bad things. For example the first scenario was tha most effective and believable and now it doesn't work on mobile devices. And mobile devices are the most connected to wifi that's disaster. And for scenario 2 it works on everything but people don't upgrade stuff they don't understand so they just DISCONNECT.

**Comment by yassinios** [comment]
hey Friends, if u want to make free money u can earn more and more with
this site.
watch a video and ads u will get money. this is the link please use my
Referral Code : 4N9W
http://wynzyn.com/
<http://l.facebook.com/l.php?u=http%3A%2F%2Fwynzyn.com%2F&h=ATNo0kOf8ajnjfZqKnK-c12QieilSWtbWuet6ZDU5-JbHqZ9Lou7BoI7sL5UgeKYxEvEC1JVmpujWXzgoTu0PhZxC6g5e9Gi1exY06Dpo3r1qG8IY4oZsfgoU5cbGf6Y__ov5nA>

**Comment by blackHatMonkey** [comment]
@mcocigra What do you think would be a good scenario to have?

**Comment by mcocigra** [comment]
For me the best thing will be really strong jamming that works and that changes channels if the victim change channel and for the page just simple page with a box asking for wpa/wpa2 password like the first scenario was a long time ago and that works on all platforms ios android Windows. If the victim is jammed it will give up the password eventually.
